### PR TITLE
Steinbeck quote added to QuoteList

### PR DESCRIPTION
**Pull request by grundydarcey**
## Description:

One quote was added to add variety to the quotes.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#1B2B58",
+        "titleBar.activeBackground": "#263D7B",
+        "titleBar.activeForeground": "#F8FAFD"
+    }
+}
```
Comment (nhcarrigan) [inline .vscode/settings.json:7]: Looks like a stray config file made it into the PR.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by grundydarcey — COMMENTED** [review]
Viewed files, thank you for your help!

**Review by grundydarcey — COMMENTED** [review]
Stray file was deleted.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**